### DISCUSSION TITLE
Should "task done" trigger "issue closed"?

### DISCUSSION THREAD

**Discussion by RickCogley** [community/community#6411] [Projects and Issues]
Maybe I'm just not understanding, but, whether I create a task from an issue, or create it newly then convert to an issue, moving the task to "done" in the kanban does nothing to the issue; not even a mention in the issue history (although, there _is_ an indicator of the task status in the project indicator on the ight). The issue will remain open. 

Perhaps this is either just not implemented, maybe it can be done via workflows later, or, maybe I'm just not getting the point... 

Can someone shed some light? 

Edit: the docs imply the statuses should be linked: 

https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects#staying-up-to-date

**Comment by cprice404** [comment]
I also have this question; we are using Milestones, and all of the milestones show 0% completed because the issues moved to the right-most column of the project board are not considered "closed".  What is the intended workflow for marking them as "closed" from the board?

**Comment by jubishop** [comment]
Yes I'd like to mark issues as Closed from my Project, have it essentially correlate to "Done"
